.github/workflows/trigger-webui.yml

```diff
@@ -41,6 +42,14 @@ jobs:
       # an unpredictable SHA; so instead, wait until COPR has a build which is
       # newer than the PR push time. This assumes that this workflow always runs earlier
       # than the COPR srpm build finishes.
+
+      - name: Install dependencies
+        run: |
+          dnf install -y git-core dnf-plugins-core gh || {
+            sleep 60
+            dnf install -y git-core dnf-plugins-core gh
+          }
```

Comment:
Could you please explain why we need to do this twice? It is just a DNF install it shouldn't be necessary to do this.

Reply:
AS mentioned in the commit, this part was just revert of the previous state. I don't know if it's needed tbh, as I copied this worflow from cockpit project.
I will remove and we can see if it flakes often.